apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1/UrlSigner.V4Signer.cs

```diff
@@ -170,10 +170,10 @@ void AddHeaders(SortedDictionary<string, string> canonicalized, IDictionary<stri
                                 continue;
                             }
                             var headerName = pair.Key.ToLowerInvariant();
-                            var value = string.Join(", ", pair.Value.Select(PrepareHeaderValue)).Trim();
+                            var value = string.Join(",", pair.Value.Select(PrepareHeaderValue)).Trim();
```

Comment:
The previous code definitely worked with integration tests. Do you want me to pull this branch to check that they still pass?